### ISSUE TITLE
Warnings have whitespace errors

### ISSUE THREAD

**Issue by hynek** [Pylons/webob#342]
I just got upgraded to WebOb 1.8.0rc1 and I got a bunch of warnings that have whitespace errors (`bywebob.acceptparse.create_accept_header`, `shimwill`).  My guess is that they’re split over multiple lines and y’all forgot to add a trailing space:

```
tests/test_functional.py::test_header_ids[/-404]
  /Users/hynek/.local/share/virtualenvs/XXX-j5QwPtpM/lib/python3.6/site-packages/webob/acceptparse.py:1077: DeprecationWarning: The MIMEAccept class has been replaced bywebob.acceptparse.create_accept_header. This compatibility shimwill be deprecated in a future version of WebOb.
    DeprecationWarning
```

It’s also not super helpful that those warnings point at webob’s code – maybe you could add a stacklevel?  I suspect those errors are coming from webtest FWIW – do they know about this? :)

**Comment by bertjwregeer** [comment]
I don't believe they are aware of the change yet, I will open a ticket with `webtest` around this change. It looks like Steve already fixed the issue in #344, which I will pull in before I release 1.8.

**Comment by bertjwregeer** [comment]
This has been fixed, and will make it to the latest WebOb 1.8 that is due to be released soon.
